sorc/ncep_post.fd/INITPOST_GFS_NETCDF_PARA.f

```diff
@@ -2594,10 +2595,13 @@ subroutine read_netcdf_3d_para(ncid,im,jm,jsta,jsta_2l,jend,jend_2u, &
       real,intent(out)   :: buf(im,jsta_2l:jend_2u,lm)
       integer            :: varid,iret,jj,i,j,l,kk
       integer            :: start(3), count(3), stride(3)
+      real,parameter     :: spval_netcdf=9.99e+20
+      real               :: fill_value
+      real,parameter     :: small=1.E-6
```

Comment:
suggest using "small" already defined in params:
use params_mod,   only: small

Reply:
@HuiyaChuang-NOAA Good point. Updated.